bazaarvoice/emodb
# new postman tetst have been created for GET uac/1/role/{role}/{id}

## What Are We Doing Here?

New postman tests for GET role /uac/1/role/{role}/{id} endpoint have been added.

## How to Test and Verify

1. Run this tests on our Jenkins with pipeline setup to run this collection file
2. Verify that tests for get role endpoint are run and passed
3. Profit

## Risk
`Low`

### Level 

`Low`

### Required Testing

 `Manual`

### Risk Summary

No problems is expected with existing code as nothing of it has been touched. Potential risk is possibility of new broken tests to be added to test suite base.

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
